### PR TITLE
changes done in the baseURL.

### PR DESCRIPTION
**Pull request by aman7870**
changed the base baseURL. imageURL is still same

### CODE REVIEW

**Comment by aman7870** [comment]
did the change in the URL and modified according to the data.

File: pages/index.js [hunk 1]
```diff
@@ -19,33 +19,32 @@ const Index = ({ membersArr, errorMsg }) => {
...
   try {
-    const res = await fetch(URL);
+    const res = await fetch(getMembersURL);
     if (res.status !== 200) {
       throw new Error(
         'There was some issues fetching the members, Please try again after some time'
       );
     }
-    let data = await res.json();
+    let { members } = await res.json();
```
Comment (sumitd94) [inline pages/index.js:32]: We are not re-assigning `{ members }` here, let's declare it as `const`

File: pages/index.js [hunk 2]
```diff
@@ -19,33 +19,32 @@ const Index = ({ membersArr, errorMsg }) => {
...
-        });
+    members.sort((a, b) => {
+      if (a.first_name < b.first_name) {
+        return -1;
       }
+      if (a.first_name > b.first_name) {
+        return 1;
+      }
+      return 0;
+    });
```
Comment (sumitd94) [inline pages/index.js:42]: we can convert it into a single line 
```javascript
members.sort((a, b) => (a.first_name > b.first_name) ? 1 : -1);
```

Neat, clean & readable.

**Review by sumitd94 — CHANGES_REQUESTED** [review]

File: jsconfig.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "experimentalDecorators": true,
```
Comment (sumitd94) [inline jsconfig.json:3]: Why is this changed?

**Review by sumitd94 — APPROVED** [review]
Thanks for the changes.

**Pull request merged**